evm/db/state.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
+
```

Comment:
Yes and no

We aren't currently testing against python 2....  Maybe we just keep this python3 only since I'm trying to deprecate python2 support around the end of this year.